resources/metainfo/wire-desktop.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.wire.WireDesktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Wire</name>
+  <summary>The most secure collaboration platform.</summary>
+  <description>
+    <p>
+      Business chats, one-click conference calls and shared documents – all protected with end-to-end encryption. Also
+      available for personal use.
+    </p>
+  </description>
+  <launchable type="desktop-id">com.wire.WireDesktop.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://wire-docs.wire.com/mustang/index/external-mobile.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://wire-docs.wire.com/mustang/index/guestroom.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://wire-docs.wire.com/mustang/index/productivity-mobile.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image>https://wire-docs.wire.com/mustang/index/productivity-ui.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://wire.com/</url>
+  <developer_name>Wire Swiss GmbH</developer_name>
+  <content_rating type="oars-1.1" />
+  <releases>
+    <release version="3.3.2872" date="2018-09-17" type="stable" />
```

Comment:
Yes, @ffflorian, unfortunately it would require extra work to update it every time for now. Also, I'm not entirely sure if you can _automatically integrate_ this in your pipeline nowadays – I meant without doing a single change in the actual build process.

Granted there are [tools](https://www.freedesktop.org/software/appstream/docs/sect-Quickstart-Translation.html) for doing these kind of things automatically, but to my knowledge most (if not all) Electron applications do not use any of that. Moreover, there is a "request" (electron-userland/electron-builder/issues/1993) on Electron Builder to add **AppStream metadata** to some/all of the supported "output formats".

That's my limited knowledge on this, and I'm trying to give you as much information as possible so you can weight the pros and cons and decide. I should have state all of that beforehand in the pull request's description :innocent:

If you want, you can wait until the Electron folks add this "natively" – the aforementioned issue `1993` in `electron-builder`. I have no problems maintaining that XML file in the Flatpak artifact for the time being.